app/views/layouts/application.html.slim

```diff
@@ -75,6 +75,25 @@ html
       / The postfix is available for all three flash names and for rendering
       / below the navigation bar or in a panel.
       /
+      / ## Flash messages of model errors
+      /
+      / When a model fails to save you may want to show its validation errors
+      / to the user. To do this in the flash you can use the `_model_errors`
+      / postfix. For exmaple:
+      /
+      /     if publisher.save
+      /       redirect_to success_url
+      /     else
+      /       flash.now[:warning_model_errors] = publisher
+      /       render action: :new
+      /     end
+      /
+      / Because the rendering expects that you pass a literal model, this
+      / utility can only be used with `flash.now`.
+      /
+      / The postfix is available for all three flash names and for rendering
+      / below the navigation bar or in a panel.
```

Comment:
This is new shared `flash.now[:warning_model_errors]` API replaces the `model_errors` partial and `content_form_error` content implementations. It should be a nice abstraction through other places we want to present validation errors in the flash box.